asllib/Parser.mly

```diff
@@ -561,7 +561,7 @@ let params_opt == { [] } | braced(clist(opt_typed_identifier))
 let access_args == bracketed(clist(typed_identifier))
 let func_args == plist(typed_identifier)
 let maybe_empty_stmt_list == stmt_list | annotated({ S_Pass })
-let func_body == delimited(BEGIN, maybe_empty_stmt_list, END)
+let func_body == delimited(BEGIN, maybe_empty_stmt_list, END; SEMI_COLON)
```

Comment:
Please make this rule optional.
When an `end` is not followed by a semicolon, please emit a warning (just once for the entire run, otherwise we'll get thousands of warnings).